proposals/drafts/elections.rst

```diff
@@ -0,0 +1,669 @@
+
+election_id
+    Reference to the OCD ``Election`` in which the contest is decided.
+
+created_at
+    Time that this object was created at in the system.
+
+updated_at
+    Time that this object was last updated in the system.
+
+sources
+    **optional**
+    **repeated**
+    List of sources used in assembling this object. Has the following properties:
+
+    url
+        URL of the resource.
+    note
+        **optional**
+        Description of what this source was used for.
+
+extras
+    Common to all Open Civic Data types, the value is a key-value store suitable for storing arbitrary information not covered elsewhere.
+
+
+BallotMeasureContest
+--------------------
+
+A subclass of ``ContestBase`` for representing a ballot measure before the voters, including summary statements on each side.
+
+id
+    Open Civic Data-style id in the format ``ocd-ballotmeasurecontest/{{uuid}}``
+
+con_statement
+    **optional**
+    Specifies a statement in opposition to the ballot measure. It does not necessarily appear on the ballot (string).
+
+effect_of_abstain
+    **optional**
+    Specifies the effect abstaining from voting on the ballot measure, i.e., whether abstaining is considered a vote against it (string).
+
+full_text
+    **optional**
+    Specifies the full text of the ballot measure as it appears on the ballot (string).
+
+passage_threshold
+    **optional**
+    Specifies the threshold of votes the ballot measure needs in order to pass. The default is a simple majority, i.e., "50% plus one vote". Other common thresholds are "three-fifths" and "two-thirds".
+
+pro_statement
+    **optional**
+    Specifies a statement in favor of the referendum. It does not necessarily appear on the ballot (string).
+
+summary_text
+    **optional**
+    Specifies a short summary of the ballot measure that is on the ballot, below the title, but above the text.
+
+type
+    **optional**
+    Enumerated among:
+
+    * ballot-measure: A catch-all for generic types of non-candidate-based contests.
+    * initiative: These are usually citizen-driven measures to be placed on the ballot. These could include both statutory changes and constitutional amendments.
+    * referendum: These could include measures to repeal existing acts of legislation, legislative referrals, and legislatively-referred state constitutional amendments.
+    * other: Anything that does not fall into the above categories.
+
+other_type
+    **optional**
+    Allows for cataloging a new type of ballot measure option, when type is specified as "other" (string).
+
+
+CandidateContest
+----------------
+
+A subclass of ``ContestBase`` for repesenting a contest among candidates competing for election to a public office.
+
+id
+    Open Civic Data-style id in the format ``ocd-candidatecontest/{{uuid}}``.
+
+filing_deadline
+    **optional**
+    Specifies the date and time when a candidate must have filed for the contest for the office (datetime).
+
+is_runoff
```

Comment:
I liked your suggestion about runoff_for_contest_id as a property here that would a) replace is_runoff (since they're redundant) and b) provide a helpful link in a clean way to the original contest.